src/plugins/intel_cpu/docs/fusing_disabling.md

```diff
@@ -0,0 +1,53 @@
+# Fusing disabling
+Fusing disabling is controlled by environment variables.
+
+The variables define conditions of the node which fusing should be disabled for.
+
+Fusing can be disabled for:
+
+- target node (node which we fuse into):
+
+  ```sh
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='>Convolution Pooling'
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='Convolution Pooling'
+  ```
+
+- fused node (node which we fuse):
+
+  ```sh
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='<Eltwise FakeQuantize'
+  ```
+
+## Disable by type
+Disable fusings for nodes with specific types:
+
+```sh
+    OV_CPU_DISABLE_FUSING_NODE_TYPE=<space_separated_list_of_types> binary ...
+```
+
+Example:
+
+```sh
+    OV_CPU_DISABLE_FUSING_NODE_TYPE='Convolution Pooling' binary ...
+```
+
+> **NOTE**: see **enum Type** in [cpu_types.h](../src/cpu_types.h) for list of the types
+
+## Disable by name
+Disable fusings for nodes with particular names:
+
+```sh
+    OV_CPU_DISABLE_FUSING_NODE_NAME=<space_separated_list_of_names> binary ...
+```
+
+Example:
+
+```sh
+    OV_CPU_DISABLE_FUSING_NODE_NAME='Convolution_123 Pooling_30' binary ...
```

Comment:
Is `>` and `<` applicable here as well?

Reply:
Yes, it is.
This special character is described in common section, I don't think we need to describe it in the sub-chapters too.